### PR TITLE
German translation

### PR DESCRIPTION
**Pull request by N3tt3rm4n**
A German translation. IMPORTANT: Please don't merge yet!

There are a few questions first: In German there are two kinds of 'you': 'Du' and 'Sie'. The first one is more informal and you use it when speaking with friends, for example, but it's also more modern imho (i. e. facebook uses 'Du'). The second one is more formal; I used 'Du', but it's up to you to decide.
Also: I kept the English Wikipedia links, should they be changed to German Wikipedia?

Have fun :smiley:

### CODE REVIEW

**Comment by Super-Baleine** [comment]
Hey HTML5 crazy man! x)

Wikipedia stuff: if the German version exists, then yea, use it please.

Formalities stuff: Use "Du", please x)

Thanks for your work man, that's really cool, tell us when you want us to merge it.
I love u 💟

File: website/en.json [hunk 1]
```diff
@@ -43,7 +43,7 @@
 		"GioRole": "Co-founder & Technical support service",
 		"GioLife": "Giovanni helps users and eventually, enterprises. He is passionate about web technologies, hardware and Android. He likes watching animes and series.",
 		"AyaRole": "Back-end developer",
-		"AyaLife": "Aya is an Eletronics & Communication Engineer whose passion is programming. She has Diploma in Open Source Web Development and works as back-end developer at Muonium. Also, she is a book author in Lap Lambert Academic Publishing. Her hobbies are music and tennis.",
+		"AyaLife": "Aya is an Electronics & Communication Engineer whose passion is programming. She has Diploma in Open Source Web Development and works as back-end developer at Muonium. Also, she is a book author in Lap Lambert Academic Publishing. Her hobbies are music and tennis.",
```
Comment (Super-Baleine) [inline website/en.json:46]: Thanks! x)

**Comment by Super-Baleine** [comment]
IT = computing/computer science, by the way x), literally: Information Technology/ies, if I've a good memory
it should be used only in the English translation.

If you can use only German words, to avoid to do germenglish language, that'd be really perfect! x)

**Comment by N3tt3rm4n** [comment]
Here is the HTML5 crazy man speaking :D
IT in German stands for "**I**nformations**t**echnologie", so that isn't a problem... Most computer interested people use germenglish language as you call it, because either there is no German word or the German word sounds cruel :smiley: For example back end in German means * *damm-damm-daaah* * Backend ^^
The Wikipedia links will come asap

Btw cool project :+1:

**Comment by N3tt3rm4n** [comment]
... and English words sound more cool in general (most German companies use English terms like "Human Resources" for their departments) ^^
What means "cek" ()? I leaved it as cek, but don't know whether to translate it or not... And what means "Mui"?

EDIT: Got it about mui^^

**Comment by Super-Baleine** [comment]
Yo,
Okay, no problem for germenglish.

CEK: content encryption key, the random key generated to proceed to a key derivation and then, to encrypt the files. x)

**Comment by Super-Baleine** [comment]
Mui is Muonium x)

Can I merge now?

**Comment by Super-Baleine** [comment]
"cool project"
Thanks for your support, we need it x)

**Review by Super-Baleine — APPROVED** [review]

**Comment by Super-Baleine** [comment]
I believe in you man,  hoping you didn't put: "Muonium is a porn website for dogs" xD

File: core/de.json [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+        "browser": "Webbrowser",
+        "browserVersion": "Browserversion",
+        "message": "Mitteilung",
+        "messageLength": "Die Mitteilung muss mindestens 50 Zeichen enthalten",
+        "form": "Fehler im Formular",
+        "sent": "Deine Mitteilung wurde erfolgreich abgesendet. Wir werden so schnell wie möglich per Email darauf antworten"
+    },
+
+	"cek": {
+		"updateErr": "Fehler: cek kann nicht aktualisiert werden",
```
Comment (Super-Baleine) [inline core/de.json]: I'm gonna update those two strings

**Comment by N3tt3rm4n** [comment]
Damn, how did you notice I wrote "Muonium is a porn website for dogs"?! :1st_place_medal: for German speaking :)

Can be merged know, maybe others will correct things I eventually made wrong...

**Comment by Super-Baleine** [comment]
Because I'm good German speaker: "Mercedes Benz", you see? xD

Okay, I'm gonna merge it then, thanks for your work!

**Pull request merged**